blaze/server/tests/test_server.py

```diff
@@ -152,9 +157,12 @@ def dont_test_compute_with_namespace():
 
 
 @pytest.fixture
-def iris_server():
+def iris_server(request):
     iris = CSV(example('iris.csv'))
     server = Server(iris)
+    ctx = server.context()
+    request.addfinalizer(lambda: ctx.__exit__(None, None, None))
+    ctx.__enter__()
```

Comment:
here, the server's application context is something that needs to exist outside of the tests; when someone runs their flask app with this blueprint they need to use the correct context.